### DISCUSSION TITLE
Hyperlink targetted selection to decorate with the underlined link

### DISCUSSION THREAD

**Discussion by cmaahs** [wezterm/wezterm#3265] [Q&A]
I'll just do a quick example 

`NAME                                                   READY   STATUS    RESTARTS   AGE`
`basic-tools-90242e                                     1/1     Running   0          109m`

In the above, I might want to do two hyperlinks, one that underlines the first NAME column object, which will be my $1 and will be used in the `format`  I'll key on `\d+/\d+` to know that this is a kubernetes POD line.  In any case, the clickable link that would be `basic-tools-90242e`, simply will `send_text` to the current pane command line, that text will be the $1.  So, on the same line, I might also want to use the same $1 match `basic-tools-90242e`, and decorate match $2, which would be `Running`, and underline `Running` as a clickable that might output `kubectl exec -it $1 -n ${NAMESPACE} -- /bin/bash`, that way I can click on the pod name to drop just that name into a command I'm building, or I can click on the Running to drop the `kubectl exec` whole command to my command line and run that quickly.  

It doesn't seem like this is a possible thing, though certainly there is a great deal to be explored and I might not have come across some magic that will make this happen... 

~cj

**Comment by wez** [comment]
There's not a lot for wezterm to unambiguously match on in that output, so it would be prone to a lot of false positives.

What I'd suggest is writing a wrapper script for that command that amends its output with [OSC 8 explicit hyperlinks](https://wezfurlong.org/wezterm/hyperlinks/#explicit-hyperlinks), then you can use a technique similar to https://github.com/wez/wezterm/discussions/3261 to make your click action work.

**Reply by cmaahs** [reply]
Thanks man, I appreciate the nudge in the right direction.
